mcs/class/corlib/corefx/GlobalizationSupport.cs

```diff
@@ -0,0 +1,11 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Globalization
+{
+    static class GlobalizationSupport
```

Comment:
Please rename this to `GlobalizationMode` with single property `static bool Invariant` to be inline with .net core type